src/components/Banner/Banner.tsx

```diff
@@ -0,0 +1,129 @@
+import * as React from 'react';
+import IReactComponentProps from '../../common/structures/IReactComponentProps';
+import classnames from 'classnames';
+import * as styles from './Banner.sass';
+import WarningSVG from '../../svg/warning';
+import CloseSVG from '../../svg/close--small';
+
+interface IProps extends IReactComponentProps {
+
+	buttonText?: string;
+	currentIndex?: number;
+	buttonOnClick?: (...params: any[]) => any;
```

Comment:
I'm seeing quite a few occurrences of `(...params: any[]) => any`. Can you add a type alias (or interface) such as `genericCallback` to reduce the duplication? I'm open to different names.

Reply:
Good idea. I'll figure something out for this.